src/components/Cards/VotingTableCard/VotingTableCard.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { VotingTableCardCell } from "../VotingTableCardCell";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+type VotingTableCardProps = {
+  name: string; // the name of the legislator
+  votesPage: object[]; // an array of Votes
+};
```

Comment:
I changed the comment styling to match yours.  

I'm new to typescript and am not sure which to use - type or interface.  Either seem to work, should I go for one or the other?

Reply:
Yeah, either work, I've seen arguments with respect to components props for both type and interface. I haven't seen one convince me to chose one over the other though.

Reply:
I've always preferred using `type` to define props because in my mind an interface is more of an OOP thing where you're defining the blueprint of a class, while conversely by using type it feels more aligned with the idea of assigning "types" to the arguments you pass into a component.

By pulling all the type information into a `type` object you're just giving a light abstraction/cleaner code to defining types on every prop that's passed into the component.

But with all that said they both work perfectly fine sooo 🤷🏻‍♂️